library/src/commonMain/kotlin/com/ioki/passenger/api/models/ApiVehicle.kt

```diff
@@ -19,5 +19,5 @@ public data class ApiVehicle(
     @SerialName(value = "storage_spaces") val storageSpaces: Int,
     val autonomous: Boolean,
     @SerialName(value = "supports_door_open_requests") val supportsDoorOpenRequests: Boolean?,
```

Comment:
This should also be non-null, right? 🤔 

And did you asked about the `door_control_available` already? 🤔 
Right now it is not delivered in case it is non-autonomous.
However, it should be set to false in that case.